apps/coscrad-frontend/src/components/resources/utils/render-aggregate-id-cell.tsx

```diff
@@ -3,5 +3,7 @@ import { Link } from 'react-router-dom';
 import { CellRenderer } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types';
 
 export const renderAggregateIdCell: CellRenderer<HasId> = ({ id }: HasId) => (
-    <Link to={id}>VIEW</Link>
+    <Link data-testid={id} to={id}>
+        VIEW
```

Comment:
I wonder if we want to use an icon for this. I've gotten negative feedback about `VIEW` being unclear or hard to look at. I realize I'm the one who put it there to start with!

Reply:
Totally agree

Reply:
I've made the "view" link into a button just to try it out.  Let me know what you think.  I shopped around for icons but couldn't find one that really means "follow this link".  The closest is the square and arrow icon that means open link in external window.

Reply:
I'll get Justin to have a look at this one.

Reply:
I'm going to add a story to the backlog for this and we can deal with it in the near future.